Algorithmia/insights.py

```diff
@@ -0,0 +1,13 @@
+import requests
+import json
+import os
+
+class Insights:
+    # Example of correct insights:
+    # {"aKey":"aValue","aKey2":"aValue2"}
+    def __init__(self, insights):
+        headers = {}
+        headers['Content-Type'] = 'application/json'
+        AQR_URL = os.getenv('ALGORITHM_QUEUE_READER_URL') or "http://localhost:9000"
+
+        requests.post(AQR_URL+"/insights", data=json.dumps(insights).encode('utf-8'), headers=headers)
```

Comment:
Something that came out of the design review: the endpoint in AQR should be prefixed with `/v1/` so we can safely make breaking API changes without breaking the `v1.X.X` client code.

I still need to modify AQR and add the prefix but that is a small change, so let's go ahead and make the change in this PR.

```
/insights --> /v1/insights
```